lava-v2-jobs-from-api.py

```diff
@@ -136,6 +141,9 @@ def main(args):
                     dtb_full = dtb
                     if arch == 'arm64':
                         dtb = str(dtb).split('/')[-1]
+                    dtb_target = dtb.partition('.dtb')[0]
+                    if targets and dtb_target not in targets:
```

Comment:
This seems to be switching the --targets arg to use dtb names instead of LAVA device type names. Admittedly only the cambridge lab used to have this set, and we don't use it any more, but why the change?

Reply:
The `--targets` option is used for bisections to only run boot tests on one device type.

It's true that this change looks wrong though, I did this a month ago to fix some issue but the bisection code now works without it so I'll drop it and update the PR.  Well spotted!